h2o-docs/src/booklets/v2_2015/source/GLMBooklet.tex

```diff
@@ -1398,6 +1399,128 @@ \subsection{Model Statistics}
+
+where
+
+\begin{itemize}
+\item $\phi$ is the dispersion parameter estimation
+\item $\mu_i$ is the prediction
+\item $y_i$ is the real value of the target variable
+\end{itemize}
+
+\emph{Note:} You must estimate the dispersion parameter. During initialization, \texttt{compute\_p\_values} and \texttt{remove\_collinear\_columns)} must be set to \texttt{True} to facilitate this estimation process. Also, \texttt{dispersion\_parameter\_method} must be set to \texttt{"pearson"}.
+
+\textbf{Binomial:}
+
+$$l \big(\mu_i (\beta); y_i, w_i \big) = w_i \big(y_i \log \{ \mu_i \} + (1-y_i) \log \{ 1-\mu_i \} \big)$$
+
+where
+\begin{itemize}
+\item $\mu_i$ is the probability of 1
+\item $y_i$ is the real value of the target variable
+\end{itemize} 
+
+
+\textbf{Quasibinomial:}
+
+\begin{itemize}
+\item If the predicted value equals $y_i$, log likelihood is 0
+\item If $\mu_i >1$ then $l(\mu_i (\beta); y_i) = y_i \log \{ \mu_i \}$
+\item Otherwise, $l(\mu_i (\beta); y_i) = y_i \log \{ \mu_i \} + (1-y_i) \log \{ 1- \mu_i \}$ where
+\begin{itemize}
+\item $\mu_i$ is the probability of 1
+\item $y_i$ is the real value of the target variable
+\end{itemize}
+\end{itemize}
+
+\textbf{Fractional Binomial:}
+
+$$l(\mu_i (\beta); y_i) = w_i \Big(y_i \times \log \big(\frac{y_i}{\mu_i} \big) + (1-y_i) \times \log \big(\frac{1-y_i}{1-\mu_i} \big) \Big)$$
+
+where
+
+\begin{itemize}
+\item $\mu_i$ is the probability of 1
+\item $y_i$ is the real value of the target variable
+\end{itemize}
+
+\textbf{Poisson:}
+
+$$l(\mu_i (\beta); y_i) = w_i \big(y_i \times \log (\mu_i) - \mu_i - \log (\Gamma (y_i +1)) \big)$$
+
+where
+
+\begin{itemize}
+\item $\mu_i$ is the prediction
+\item $y_i$ is the real value of the target variable
+\end{itemize}
+
+\textbf{Negative Binomial:}
+
+$$l(\mu_i (\beta); y_i, w_i) = y_i \log \big(\frac{k \mu}{w_i} \big) - \big(y_i + \frac{w_i}{k} \big) \log \big(1 + \frac{k \mu}{w_i} \big) + \log \Big(\frac{\Gamma \big( y_i = \frac{w_i}{k} \big)} {\Gamma (y_i +1) \Gamma \big(\frac{w_i}{k}\big)} \Big)$$
+
+where
+
+\begin{itemize}
+\item $\mu_i$ is the prediction
+\item $y_i$ is the real value of the target variable
+\item $k = \frac{1}{\phi}$ is the dispersion parameter estimation
+\end{itemize}
+
+\emph{Note:} You must estimate the dispersion parameter. During initialization, \texttt{compute\_p\_values} and \texttt{remove\_collinear\_columns)} must be set to \texttt{True} to facilitate this estimation process. Also, \texttt{dispersion\_parameter\_method} must be set to \texttt{"ml"}. Since texttt{dispersion\_parameter\_method} only works without regularization, you disable regularization by setting \texttt{lambda} equal to an array of \texttt{0.0}.
+
+\textbf{Gamma:}
+
+$$l(\mu_i (\beta); y_i, w_i) = \frac{w_i}{\phi} \log \big( \frac{w_i y_i}{\phi \mu_i} \big) - \frac{w_i y_i}{\phi \mu_i} - \log (y_i) - \log \big(\Gamma \big(\frac{w)i}{\phi} \big) \big)$$
```

Comment:
Please change
$$l(\mu_i (\beta); y_i, w_i) = \frac{w_i}{\phi} \log \big( \frac{w_i y_i}{\phi \mu_i} \big) - \frac{w_i y_i}{\phi \mu_i} - \log (y_i) - \log \big(\Gamma \big(\frac{w)i}{\phi} \big) \big)$$

to

$$l(\mu_i (\beta); y_i, w_i) = \frac{w_i}{\phi} \log \big( \frac{w_i y_i}{\phi \mu_i} \big) - \frac{w_i y_i}{\phi \mu_i} - \log (y_i) - \log \big(\Gamma \big(\frac{w_i}{\phi} \big) \big)$$